FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2ListItemActivity.kt

```diff
@@ -760,15 +752,6 @@ fun RightViewText(text: String) {
     return Text(text = text)
 }
 
-fun onClick(text: String, coroutineScope: CoroutineScope, scaffoldState: ScaffoldState) {
-    scaffoldState.snackbarHostState.currentSnackbarData?.dismiss()
-    coroutineScope.launch {
-        val result = scaffoldState.snackbarHostState.showSnackbar(
-            message = "Clicked on $text",
-            actionLabel = "Close"
-        )
-        when (result) {
-            SnackbarResult.ActionPerformed -> scaffoldState.snackbarHostState.currentSnackbarData?.dismiss()
-        }
-    }
+fun onClick(text: String, context: Context) {
```

Comment:
This function has been added in DemoUtils. Remove this & refer the DemoUtils one.
Code Pointer :
https://github.com/microsoft/fluentui-android/blob/master/FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/util/DemoUtils.kt#L22